ports/esp32/boards/ARDUINO_NANO_ESP32/manifest.py

```diff
@@ -0,0 +1,12 @@
+include("$(PORT_DIR)/boards/manifest.py")
+
+# Networking
+require("bundle-networking")
```

Comment:
This is already required by the `boards/manifest.py` (included above), so can be removed.

Reply:
Okay, removed.